### DISCUSSION TITLE
Decomplle android APK with APkTool for analysis (-a) not working?

### DISCUSSION THREAD

**Discussion by Uzver123** [iBotPeaches/Apktool#3408] [Q&A] [answered]
On homepage you list **_Not interested in rebuilding? Just attach -a to any command. Apktool will do its best to rip apart the resources and manifest for easy inspection._**

However -a argument is not listed when i **apktool --help** and using **-a** option does not work, **apktool.bat d myfile.apk -a** getting error **Missing argument for option -a**

**Comment by iBotPeaches** [comment, marked as answer]
Looks like I butchered the docs - that mode (analysis-mode) is `-m / --match-original`.

Pushed update to docs - https://github.com/iBotPeaches/Apktool/commit/4f2e95384d2a0fb0e207a916c5f7b324304e205c

Thanks

**Reply by Uzver123** [reply]
Thank's perhaps also add description to --help for application.

**Reply by iBotPeaches** [reply]
It's under `--advanced`, the regular help menu is for basic operations.
